parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -380,14 +380,14 @@ createKubeManifestDir() {
 }
 
 writeKubeConfig() {
-    KUBECONFIGDIR=/home/$ADMINUSER/.kube
-    KUBECONFIGFILE=$KUBECONFIGDIR/config
-    mkdir -p $KUBECONFIGDIR
-    touch $KUBECONFIGFILE
-    chown $ADMINUSER:$ADMINUSER $KUBECONFIGDIR
-    chown $ADMINUSER:$ADMINUSER $KUBECONFIGFILE
-    chmod 700 $KUBECONFIGDIR
-    chmod 600 $KUBECONFIGFILE
+    local DIR=/home/$ADMINUSER/.kube
+    local FILE=$DIR/config
+    mkdir -p $DIR
+    touch $FILE
+    chown $ADMINUSER:$ADMINUSER $DIR
+    chown $ADMINUSER:$ADMINUSER $FILE
```

Comment:
This could be one command:
`chown $ADMINUSER:$ADMINUSER $DIR $FILE`

Reply:
Oh I am so on this one.